lib/view_component/base.rb

```diff
@@ -243,6 +245,25 @@ def collection_parameter_name
 
       private
 
+      def compiled_template(file_path)
```

Comment:
Note: Moving this here to preserve the (maybe previously unintentional, but intentional from me) alphabetical order of the private methods